packages/headless/src/controllers/knowledge/generated-answer/headless-answerapi-generated-answer.ts

```diff
@@ -159,7 +159,6 @@ export function buildAnswerApiGeneratedAnswer(
       return {
         ...getState().generatedAnswer,
         answer: answerApiState?.answer,
-        citations: answerApiState?.citations ?? [],
```

Comment:
I was wondering how the citations could be displayed after this change. But that's because `stream-answer-api.ts` is taking care of dispatching the `updateCitations`, which then sets `citations` correctly in the `generatedAnswer` state section. It makes more sense to me now.

Good job finding that!